.github/config/goreleaser.yaml

```diff
@@ -97,11 +97,12 @@ nfpms:
     vendor: "Open Component Model"
     homepage: "https://ocm.software/"
     maintainer: "OCM Project <open-component-model@noreply.github.com>"
-    description: "The OCM CLI makes it easy to create component versions and embed them in build processes."
+    description: |
+      "The OCM CLI is used to create and maintain component versions.
+      It can be used to easily embed OCM related actions in CICD processes."
     license: "Apache-2.0"
     formats:
       - deb
-      - archlinux
```

Comment:
why do we remove archlinux?

Reply:
because the packaging happens in another action that Hilmar just works on and that you commented on already :-) I'm about to move out Debian, too.